src/eureka-client.js

```diff
@@ -204,18 +205,16 @@ export class Eureka {
     if (!registry) {
       throw new Error('Unable to transform empty registry');
     }
-    if (registry.length) {
```

Comment:
Why is that? I have only seen registry as an object.
El El Sun, Aug 23, 2015 a la(s) 1:27 PM, Jacob Quatier <
notifications@github.com> escribió:

> In src/eureka-client.js
> <https://github.com/jquatier/eureka-js-client/pull/16#discussion_r37710180>
> :
>
> > @@ -204,18 +205,16 @@ export class Eureka {
> >      if (!registry) {
> >        throw new Error('Unable to transform empty registry');
> >      }
> > -    if (registry.length) {
>
> You know I added this for a reason, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jquatier/eureka-js-client/pull/16/files#r37710180>.
>